From bd4e1fdb05267f60b556c200754f6a1702bde3ee Mon Sep 17 00:00:00 2001
From: Marinus Schraal <mschraal@gnome.org>
Date: Thu, 22 Oct 2020 00:31:35 +0200
Subject: [PATCH] grltrackerwrapper: Remove quotes for IN filter

With the move to Tracker3 the parser no longer handles quoted arguments
to the IN operater.
---
 gnomemusic/grilowrappers/grltrackerwrapper.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gnomemusic/grilowrappers/grltrackerwrapper.py b/gnomemusic/grilowrappers/grltrackerwrapper.py
index 87189e3b9..48b1ad724 100644
--- a/gnomemusic/grilowrappers/grltrackerwrapper.py
+++ b/gnomemusic/grilowrappers/grltrackerwrapper.py
@@ -355,7 +355,7 @@ class GrlTrackerWrapper(GObject.GObject):
                     break
 
     def _song_media_query(self, media_ids):
-        media_ids = str(media_ids)[1:-1]
+        media_ids = str(media_ids)[1:-1].replace("'", "")
 
         query = """
         SELECT
-- 
GitLab

